daemon/global_wayland.h

```diff
@@ -0,0 +1,97 @@
+/*
+** Copyright 2017 Intel Corporation
+**
+** Licensed under the Apache License, Version 2.0 (the "License");
```

Comment:
One major concern here. Why include so many function prototypes which are not used?

Reply:
I guess this file is copied as is from IAS location. @satyeshwar  Can comment on this ?